docs/introduction/quick-start.md

```diff
@@ -0,0 +1,300 @@
+# Quick Start
+
+This guide will help you quickly get started with **UniCore**. You will:
+
+1. Deploy a UniCore instance locally using Docker Compose.
+2. Issue a Credential to a user.
+3. Verify that Credential.
+
+### Prerequisites
+* [Docker](https://docs.docker.com/engine/install/)
+* **UniMe** app or any other Identity Wallet that supports the following standards:
+  * **OpenID for Verifiable Credential Issuance** Working Group Draft 13
+  * **OpenID for Verifiable Presentations** Working Group Draft 20
+  * **Self-Issued OpenID Provider v2** Working Group Draft 13
+
+---
+
+## Download UniMe
+
+Download the UniMe app on your mobile device:
+
+* [App Store](https://testflight.apple.com/join/cNfBowKB)
+* [Google Play Store](https://play.google.com/store/apps/details?id=com.impierce.identity_wallet&pli=1)
```

Comment:
I think you can safely remove that last parameter: `&pli=1`

Reply:
fixed